app/controllers/idv/account_verified_cta_visited_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Idv
+  class AccountVerifiedCtaVisitedController < ApplicationController
+    before_action :disable_caching
+    before_action :validate_redirect_url_exists
+
+    rescue_from ActionController::ActionControllerError, with: :render_not_acceptable
+
+    def show
+      redirect_to(redirect_url, allow_other_host: true)
```

Comment:
This is an [open redirect vulnerability](https://cheatsheetseries.owasp.org/cheatsheets/Unvalidated_Redirects_and_Forwards_Cheat_Sheet.html).

http://localhost:3000/redirect/return_to_sp/account_verified_cta?redirect_url=https://example.com

Reply:
Maybe we can parameterize the service provider issuer, and do the lookup for the redirect URL within the application using that?

Reply:
This is true and I thought to include a lookup for the service provider return URL.

However, I'm also fairly sure a service provider doesn't always have a return URL or an issuer string, but I may be misremembering.

I'll see how I can handle that case.

Reply:
There will always be an issuer string, but there will not necessarily be a return URL

Reply:
I wonder if the fix for "there will not necessarily be a return URL" is to check for that at the point that we emit links to this controller in the first place, and never generate a redirect link to nowhere. Because a link here with no URL is effectively saying, "OK, now redirect the user to nowhere!" There really is no sensible thing we can do in that case.

There still remains the small chance that we would remove the return URL after emailing the link, but I think we should _expect_ breakage if we make such a change.

Reply:
Yeah, I'd agree that it'd be best to not try to include it when the email is sent

Reply:
This has been fixed with the latest changes.